Classes/Issues/Comments/IssueCommentSectionController.swift

```diff
@@ -75,6 +76,25 @@ AttributedStringViewIssueDelegate {
         return AlertAction(AlertActionBuilder { $0.rootViewController = weakSelf?.viewController })
             .share([url], activities: [TUSafariActivity()]) { $0.popoverPresentationController?.sourceView = sender }
     }
+    
+    func deleteAction() -> UIAlertAction? {
+        guard object?.viewerCanDelete == true, let number = object?.number else { return nil }
+        
+        return AlertAction.delete({ [weak self] _ in
+            let title = NSLocalizedString("Are you sure?", comment: "")
+            let message = NSLocalizedString("Deleting this comment is irreversible, do you want to continue?", comment: "")
+            let alert = UIAlertController.configured(title: title, message: message, preferredStyle: .alert)
+            
+            alert.addActions([
+                AlertAction.cancel(),
+                AlertAction.yes({ [weak self] _ in
```

Comment:
I think this should be a `delete` action, not `yes`.

Reply:
I did this initially then changed it based on how we handle other confirmation alerts, namely sign out of all accounts.

Happy to change